README.md

```diff
@@ -18,6 +18,7 @@ In addition to this README, the following is available in the [flutter plugin Gi
 * [`doc`] - Root directory for all documentation.
 * [`example`] - Root directory of the Flutter sample app (with walkthrough).
 * [Getting started with the example app]
+* [Override the version of iOS and Android In-App Payments SDK]
```

Comment:
The title of the linked doc is "Versioning Strategy".  This list item should match the title of the linked doc.